huey_monitor/humanize.py

```diff
@@ -41,16 +41,19 @@ def format_sizeof(num, suffix='', divisor=1000):
     return f'{num:3.1f}Y' + suffix
 
 
-def percentage(num, total):
+def percentage(num, total) -> str | None:
     """
     >>> percentage(25, 100)
     '25%'
     >>> percentage(33.333, 100)
     '33%'
+    >>> percentage(123, 0)
+    None
     """
-    frac = num / total
-    percentage = frac * 100
-    return f'{percentage:.0f}%'
+    if total:
+        frac = num / total
+        percentage = frac * 100
+        return f'{percentage:.0f}%'
```

Comment:
this implicit return looks ugly to me, would rather go `if not total: return None` at the start.

Reply:
updated